cmd/zarb/tx/common.go

```diff
@@ -0,0 +1,36 @@
+package tx
+
+import (
+	"strings"
+
+	"github.com/zarbchain/zarb-go/cmd"
+	"github.com/zarbchain/zarb-go/keystore/key"
+	"github.com/zarbchain/zarb-go/tx"
+	zarb "github.com/zarbchain/zarb-go/www/grpc/proto"
+)
+
+func signAndPublish(trx *tx.Tx, keyfile, auth string, grpcClient zarb.ZarbClient) {
+	//sign transaction
+	k, err := key.DecryptKeyFile(keyfile, auth)
+	if err != nil {
+		cmd.PrintErrorMsg("Couldn't retrieve Key: %v", err)
+		return
+	}
+	k.ToSigner().SignMsg(trx)
+
+	cmd.PrintWarnMsg("you are about to publish:")
+	cmd.PrintJSONObject(trx)
```

Comment:
better print the raw bytes for user that want to send the raw transaction.

use `cmd.PrintLine()` for line separator.